public/manifest.json

```diff
@@ -1,25 +1,24 @@
 {
   "short_name": "React App",
-  "name": "Create React App Sample",
+  "name": "Create React App",
```

Comment:
You can change the name and short name with your ecommece name

Reply:
Yes. I will do that. Thankyou for pointing it.